cli/src/api/mod.rs

```diff
@@ -281,6 +298,20 @@ impl PhylumApi {
         self.get(endpoints::get_package_status(&self.api_uri, pkg))
             .await
     }
+
+    /// Get all groups the user is part of.
+    pub async fn get_groups_list(&mut self) -> Result<ListUserGroupsResponse> {
+        self.get(endpoints::group_list(&self.api_uri)).await
+    }
+
+    /// Get all groups the user is part of.
+    pub async fn create_group(&mut self, group_name: &str) -> Result<CreateGroupResponse> {
```

Comment:
Due to the way our functions work, *some* return type is expected here though and I'd personally rather not use a dummy one like `serde_json::Value`? This would be the first place where we don't actually check the result afaik.

Reply:
The call to `self.post()` needs a return type, but [`IgnoredAny`](https://docs.serde.rs/serde/de/struct.IgnoredAny.html) would work there.

But I understand the concern about matching what we do for other endpoints. I'll save this for a later discussion.